parm/config/gfs/config.resources

```diff
@@ -390,7 +390,7 @@ case ${step} in
       "C1152" | "C768")
         layout_x=8
         layout_y=8
-        walltime="00:45:00"
+        walltime="01:45:00"
```

Comment:
This is JEDI atmospheric analysis.  Is this change intended for GFSv17 cycled testing?

Reply:
This is the aerosol job I think? We were originally running with aerosols and had to increase the time.  We did make a configuration change which reduced the time.  All the same this could probably be left out of this PR. 

@CoryMartin-NOAA

Reply:
Yes this change doesn't have to happen here, I'll do updated resource requests as part of the GCAFS cycling PR